src/components/DataCollector/utils.js

```diff
@@ -50,8 +50,23 @@ const isSupportedAnchorElement = element => {
   return false;
 };
 
+const isDownloadLink = (downloadLinkQualifier, linkUrl, clickedObj) => {
+  const re = new RegExp(downloadLinkQualifier);
+  return clickedObj.download ? true : re.test(linkUrl.toLowerCase());
+};
+
+const isExitLink = (window, linkUrl) => {
+  const currentHostname = window.location.hostname;
```

Comment:
Is `window.location.hostname` guaranteed to be lowercase already?

Reply:
I read that it was but I can probably throw in a toLowerCase to be safe